101-privatelink-endpoint-sql/azuredeploy.json

```diff
@@ -0,0 +1,295 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "administratorLogin": {
+      "type": "string",
+      "metadata": {
+        "description": "The administrator username of the SQL logical server"
+      }
+    },
+    "administratorLoginPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The administrator password of the SQL logical server."
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "secureString",
+      "metadata": {
+        "description": "Password for the Virtual Machine. The password must be at least 12 characters long and have lower case, upper characters, digit and a special character (Regex match)"
+      }
+    },
+    "VmSize": {
+        "type": "string",
+        "defaultValue": "Standard_D2_v2",
+        "metadata": {
+            "description": "The size of the VM"
+        }
+    },    
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "variables": {
+    "vnetName": "myVirtualNetwork",
+    "vnetAddressPrefix": "10.0.0.0/16",
+    "subnet1Prefix": "10.0.0.0/24",
+    "subnet1Name": "mySubnet",
+    "sqlServerName": "[concat('sqlserver', uniqueString(resourceGroup().id))]",
+    "databaseName": "[concat(variables('sqlServerName'),'/sample-db')]",
+    "privateEndpointName": "myPrivateEndpoint",
+    "privateDnsZoneName": "privatelink.database.windows.net",
```

Comment:
this looks to be specific for public Azure - if it's not, then use something generic so it doesn't look "wrong" for Azure Government

Reply:
Hi this is the Private DNS Zone name that is suggested for the Azure SQL private endpoint
https://docs.microsoft.com/en-us/azure/private-link/private-endpoint-dns#azure-services-dns-zone-configuration

Reply:
I'm not sure if that means the doc should be updated or we expect sovereign clouds top point to public Azre?

Reply:
I have asked the product group and this Private DNS Zone name is for public Azure , so Im going to limit this quick start for public cloud

Reply:
ok, I'm restoring the badge so we can communicate that it's not supported in FF